app/index.html

```diff
@@ -558,7 +558,9 @@ <h1>Tell us about your <span class="underline">idea</span></h1>
       <div class="" style="margin: 20px 0;">
         <a href="https://clutch.co/profile/therefore-interactive" target="_blank"><img
           src="images/web_developers_canada_2017.png" alt="Clutch Rated Top Development Design Agency"></a>
-        <img src="images/acquia_foundation_partner.png" alt="Acquia Foundation Partner" class="acquia-logo">
+        <!--<img src="images/acquia_foundation_partner.png" alt="Acquia Foundation Partner" class="acquia-logo">-->
+        <img src="images/pantheon.jpeg" alt="Netlify" class="acquia-logo">
```

Comment:
@chandantherefore The alt tags for these two images just need to be swapped, got mixed up.